Docs review updated 2024-03-monthly.md (#4)
cardano-scaling/website: monthly/2024-03-monthly.md

Having full `Conway` era support, including `PlutusV3`, on the Hydra layer 2 already prepared on a branch allows early adopting users from the community to explore and use this already. Modulo-P has been on the forefront here by developing zero-knowledge hybrid DApps and wrapping the Hydra branch up in this [example repository](https://github.com/Modulo-P/hydra-node-plutusv3).

:::info
A full-on `Conway` `hydra-node` can be used on `SanchoNet` already and the Hydra protocol scripts did not change from `master` / the upcoming `0.16.0`. However, note that the [Hydra explorer](explorer.hydra.family) shows such `Conway`-built heads only as `Initializing`. More details about why can be found in this [pull request](https://github.com/input-output-hk/hydra/pull/1373).
:::

### Streaming plugins

SundaeLabs is working on a Hydra-based layer two protocol (`gummiworm`) that requires the usage of `hydra-node` components in a different way than the default setup. As part of this development, SundaeLabs submitted a Fund10 Catalyst proposal to enhance the Hydra node by adding features that would facilitate not only their work but also the work of everyone building on Hydra.

> The team has been working on a plugin architecture to enable the infiltration and exfiltration of events, which enables hygienic forks of Hydra that feature additional connections to the outside world. SundaeLabs, for instance, is interested in consuming the Hydra event stream via Amazon Kinesis, and storing transaction archives on S3 for later reference.

Reviewer: @oduameh Here I was referencing the SundaeLabs team with `they`. Should we be more inclusive and call anyone working on Hydra or Mithril as `the team`?

> The team has been working on a plugin architecture to enable the infiltration and exfiltration of events, which enables hygienic forks of Hydra that feature additional connections to the outside world. SundaeLabs, for instance, is interested in consuming the Hydra event stream via Amazon Kinesis, and storing transaction archives on S3 for later reference.

Reviewer 2: ```suggestion
SundaeLabs have been working on a plugin architecture to enable the infiltration and exfiltration of events, which enables hygienic forks of Hydra that feature additional connections to the outside world. They are interested in consuming the Hydra event stream via Amazon Kinesis and storing transaction archives on S3 for later reference.
```

After an initial design phase resulting in [ADR29](https://hydra.family/head-protocol/adr/29/), they contributed the event source and sink abstractions, and refactored the Hydra node to use these new extension points together with core contributors. The `EventSource` and `EventSink` interfaces, along with the configurable list of `eventSinks` in the main `HydraNode` handle, allow to extend the `hydra-node` with any external system. It also paves the way for more scalable persistence methods on the mainline Hydra node, and might even allow refactoring the existing API server as an `EventSink`.

## Community

### Mithril office hours

> The Catalyst project of TxPipe on Mithril has been completed by the team, and they want to continue building on this initial maintenance work in a more open format. To achieve this, the team is transforming their alignment calls into a public weekly office hour where they invite the community to join and discuss concrete pull requests, issues, and questions related to Mithril. 

Reviewer: @oduameh I specifically chose a passive tone here and `TxPipe` was the subject of completing the project (it's theirs). It is a related point as I commented above. In both cases, I wanted to specifically say that SundaeLabs/TxPipe has been contributing (to underline the fact that it's not only IOG people working on this).

How should we make this align with this all `The team`-speak?

> The Catalyst project of TxPipe on Mithril has been completed by the team, and they want to continue building on this initial maintenance work in a more open format. To achieve this, the team is transforming their alignment calls into a public weekly office hour where they invite the community to join and discuss concrete pull requests, issues, and questions related to Mithril. 

Reviewer 2: ```suggestion
As the Catalyst project of TxPipe on Mithril has been completed, Hydra and Mithril projects aim to continue building on this initial maintenance work in a more open format. To achieve this, the scaling teams are transforming their alignment calls into a public weekly office hour where they invite the community to join and discuss concrete pull requests, issues, and questions related to Mithril. 
```
